c++/src/Writer.cc

```diff
@@ -45,11 +45,11 @@ namespace orc {
     std::string timezone;
     WriterMetrics* metrics;
     bool useTightNumericVector;
-    uint64_t outputBufferCapacity;
+    uint64_t memoryBlockSize;
 
     WriterOptionsPrivate() : fileVersion(FileVersion::v_0_12()) {  // default to Hive_0_12
       stripeSize = 64 * 1024 * 1024;                               // 64M
-      compressionBlockSize = 64 * 1024;                            // 64K
+      compressionBlockSize = 256 * 1024;                           // 256K
```

Comment:
Why do we make a change here?

Reply:
We should not change the default behavior if unnecessary.

Reply:
I will cancel these changes